pkgs/by-name/na/nats-surveyor/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildGoModule, fetchFromGitHub, }:
+buildGoModule rec {
+  pname = "nats-surveyor";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "nats-io";
+    repo = "nats-surveyor";
+    rev = "v${version}";
+    hash = "sha256-7HJgSwsB7m1Y6U2+ZMQ0JHHdLge5v7wLjH5chlhB0F0=";
+  };
+
+  vendorHash = "sha256-NjwUiommvCy5arr1QCcDjqnxuLwN3fsj3C1O6hZL2jw=";
+
+  meta = with lib; {
+    homepage = "https://github.com/nats-io/nats-surveyor";
+    description = "NATS Monitoring, Simplified.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

> Not end with a period (or any punctuation for that matter).

`meta.description` should not end with a period.